content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -135,149 +150,66 @@ net stop /y was
-  ENV CORECLR_ENABLE_PROFILING=1
-  ENV CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
-  ENV CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
-  ENV DD_INTEGRATIONS=/opt/datadog/integrations.json
-  ENV DD_DOTNET_TRACER_HOME=/opt/datadog
-
-  # Start your application
-  CMD ["dotnet", "example.dll"]
-  ```
-
-#### `systemctl` (per service)
+##### Application restart
 
-When using `systemctl` to run .NET applications as a service, you can add the required environment variables to be loaded for a specific service.
+For standalone applications, manually restart the application as you normally would.
 
-1. Create a file called `environment.env` containing:
+{{% /tab %}}
 
-    ```ini
-    CORECLR_ENABLE_PROFILING=1
-    CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
-    CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
-    DD_INTEGRATIONS=/opt/datadog/integrations.json
-    DD_DOTNET_TRACER_HOME=/opt/datadog
-    # any other environment variable used by the application
-    ```
-2. In the service's configuration file, reference this as an [`EnvironmentFile`][1] in the service block:
+{{% tab "NuGet" %}}
 
-    ```ini
-    [Service]
-    EnvironmentFile=/path/to/environment.env
-    ExecStart=<command used to start the application>
-    ```
-3. Restart the .NET service for the environment variable settings to take effect.
+##### Required environment variables
 
-#### `systemctl` (all services)
+```
+CORECLR_ENABLE_PROFILING=1
+CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+CORECLR_PROFILER_PATH=<System-dependent path>
+DD_INTEGRATIONS=<APP_DIRECTORY>/integrations.json
+DD_DOTNET_TRACER_HOME=<APP_DIRECTORY>
+```
 
-<div class="alert alert-warning">
-  <strong>Note:</strong> The .NET runtime tries to load a profiler into <em>any</em> .NET process that is started with these environment variables set. You should limit instrumentation to only the applications that need to be traced. <strong>Don't set these environment variables globally as this causes <em>all</em> .NET processes on the host to load the profiler.</strong>
-</div>
+The value for the `CORECLR_PROFILER_PATH` environment variable will vary based on the system where the application is running. See the following table for the appropriate path:
 
-When using `systemctl` to run .NET applications as a service, you can also set environment variables to be loaded for all services run by `systemctl`.
+Operating System and Process Architecture | CORECLR_PROFILER_PATH Value
+------------------------------------------|----------------------------
+Alpine Linux x64 | `<APP_DIRECTORY>/datadog/linux-musl-x64/Datadog.Trace.ClrProfiler.Native.so`
+Linux x64        | `<APP_DIRECTORY>/datadog/linux-x64/Datadog.Trace.ClrProfiler.Native.so`
+Linux ARM64      | `<APP_DIRECTORY>/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so`
+Windows x64      | `<APP_DIRECTORY>/datadog/win-x64/Datadog.Trace.ClrProfiler.Native.dll`
```

Comment:
The `/` won't be an issue on Windows will it (i.e. should it be `\`? 🤔 If not, then it's probably best to leave as-is, as it avoids potential escaping issues (i.e. `\\`)

Reply:
I think you're right that `/` may be an issue. Changing it to `\` should be better, even if there is a potential escaping issue

Reply:
Updated with `\` in the latest set of commits